### PR TITLE
Broker 461

### PR DESCRIPTION
**Pull request by bgavrilMS**
Explicit support broker for .net 461 as requested by VS installer and VS feedback teams

### CODE REVIEW

**Review by pmaytak — APPROVED** [review]
- Please create an issue.
- Merge after verified by VS?
- Triggered [release bits here](https://identitydivision.visualstudio.com/IDDP/_build/results?buildId=963026&view=results)

File: src/client/Microsoft.Identity.Client.Broker/Microsoft.Identity.Client.Broker.csproj [hunk 1]
```diff
@@ -42,18 +45,8 @@
     <ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(TargetFramework)' == '$(TargetFrameworkNetDesktop)'">
```
Comment (neha-bhargava) [inline src/client/Microsoft.Identity.Client.Broker/Microsoft.Identity.Client.Broker.csproj:45]: Why are these removed?

File: src/client/Microsoft.Identity.Client.Broker/Microsoft.Identity.Client.Broker.csproj [hunk 2]
```diff
@@ -1,12 +1,15 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+
```
Comment (neha-bhargava) [inline src/client/Microsoft.Identity.Client.Broker/Microsoft.Identity.Client.Broker.csproj:2]: Nit: extra line

**Review by neha-bhargava — APPROVED** [review]

File: src/client/Microsoft.Identity.Client.Broker/Microsoft.Identity.Client.Broker.csproj (hunk 1 above)
Comment (bgavrilMS) [reply src/client/Microsoft.Identity.Client.Broker/Microsoft.Identity.Client.Broker.csproj:45]: For SDK projects, I think we should rely on the precompilation constants which are already defined by .NET, e.g. `if NET45_OR_HIGHER`. The whole `DESKTOP`, `WINDOWS_APP` naming is unfortunate. Can add back though, not a problem.

**Review by SameerK-MSFT — APPROVED** [review]

**Review by gladjohn — APPROVED** [review]

**Comment by bgavrilMS** [comment]
@gladjohn - is this still required? If not, please feel free to close it.
